### PR TITLE
Fix integration tests and add seperate webpack build

### PR DESCRIPTION
**Pull request by WardenGnaw**
This PR cleans up unused gulp tasks and npm run commands. 

Fixing how tests are ran. Need to compile tests with TSC after webpack.

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -14,4 +14,22 @@ jobs:
     pool:
       vmImage: 'ubuntu-16.04'
 
-- template: Extension/jobs/build.windows.yml
\ No newline at end of file
+- template: Extension/jobs/build.windows.yml
+
+- job: WebackCompile
```
Comment (sean-mcmanus) [inline azure-pipelines.yml]: Is this supposed to be "WebpackCompile"?

**Comment by sean-mcmanus** [comment]
Your comment appears cutoff? "webpack build will be used for..."?

**Review by sean-mcmanus — APPROVED** [review]

**Comment by pieandcakes** [comment]
@sean-mcmanus your tests are still not passing with his change.

**Comment by sean-mcmanus** [comment]
@pieandcakes That's fine -- they weren't passing before -- we can fix that afterwards, right?

**Comment by pieandcakes** [comment]
@sean-mcmanus ok.. but I thought the point of this PR was to fix your tests.

**Comment by sean-mcmanus** [comment]
Well, it's failing with:
`
nvm is not compatible with the npm config "prefix" option: currently set to "/usr/local"

Run `npm config delete prefix` or `nvm use --delete-prefix v6.17.1 --silent` to unset it.

Error: Error: Cannot find module '/Users/vsts/agent/2.150.3/work/1/s/Extension/out/test/integrationTests/languageServer'
`
so maybe that's related to this change?

**Comment by sean-mcmanus** [comment]
Should the integration tests be removed from the build.*.yml?
`
  - script: 'node node_modules/vscode/bin/test'
    displayName: 'Run languageServer integration tests'
`

**Comment by WardenGnaw** [comment]
@sean-mcmanus The tests are now hard failing for you to fix.

**Comment by WardenGnaw** [comment]
Moved branch to https://github.com/Microsoft/vscode-cpptools/tree/dev/waan/fixTestRun

**Pull request closed without merging**